src/data/pokemon/species_info.h

```diff
@@ -23894,7 +23896,7 @@ const struct SpeciesInfo gSpeciesInfo[] =
         .baseSpAttack  = 80,
         .baseSpDefense = 55,
         .types = { TYPE_WATER, TYPE_WATER},
-        .catchRate = 25,
+        .catchRate = 190,
```

Comment:
The catch rate seems to be 75

Reply:
The data I posted earlier has it at a catch rate of 190.

Reply:
Also, Bulbapedia has the catch rate as 190 as well.